guild.go

```diff
@@ -673,26 +499,17 @@ func (c *Client) ModifyGuildRolePositions(guild objects.Snowflake, params []*Mod
 		return nil, err
 	}
 
-	res, err := c.request(&request{
-		method:      http.MethodPatch,
-		path:        fmt.Sprintf(GuildRolesFmt, guild),
-		contentType: JsonContentType,
-		body:        data,
-	})
-	if err != nil {
-		return nil, err
-	}
+	roles := []*objects.Role{}
+	err = NewRequest().
+		Method(http.MethodPatch).
+		Path(fmt.Sprintf(GuildRolesFmt, guild)).
+		ContentType(JsonContentType).
+		Body(data).
+		Expect(http.StatusOK).
+		Bind(roles).
```

Comment:
We need to pass a pointer to the slice here, otherwise it will automatically return an error.